module1/lessons/nested_collections.md

```diff
@@ -90,10 +104,73 @@ pizza_toppings = {veggies: ["green peppers", "jalapeño", "mushrooms"],
 * How can I access the element `"pineapple"`
 * How can I add the element `"olives"` to the key `"veggies"`
 
-## Checks for Understanding
+## Nested Collections in the 'real world'
+
+What about on the job? How often will you need to work with nested collections _really_? 
+
+Take a look at this [ifixit.com documentation](https://www.ifixit.com/api/2.0/doc/Stories#get-a-list-of-stories) (these docs are for working with an external API, which you'll learn about in a future module at Turing). We're specifically interested in the 'Response' section from the `GET /stories` dropdown.
+
+
+<section class="call-to-action">
+### Discuss
+
+* How could I describe the data in this section of the documentation?
+* How can I return the 'id' value for an image?
+</section>
+
+Even outside of API responses, you will encounter deeply nested data. For example, this JavaScript object:
+
+```javascript
+var meals = {
+  breakfast: { time: 9, dish: { name: 'scrambled eggs', ingredients: ['eggs', 'butter'] } },
+  lunch: { time: 12, dish: { name: 'salad', ingredients: ['kale', 'cherry tomatoes'] } },
+  dinner: { time: 6, dish: { name: 'burrito', ingredients: ['tortilla', 'tofu', 'salsa verde'] } }
+}
+```
+
+In JavaScript we can use bracket notation on this object like we would for a Ruby hash, or we can use dot notation to <strong>chain</strong> multiple properties, to dig into the values we want.
+
+```javascript
+meals.breakfast.time
+// 9
+
+meals.dinner.dish.name
+// 'burrito'
+```
+
+What would `meals` look like if we converted it into a nested collection in Ruby?
+
+<section class="dropdown">
+### Ruby version 
+
+```ruby
+meals = {
+  breakfast: { time: 9, dish: { name: 'scrambled eggs', ingredients: ['eggs', 'butter'] } },
+  lunch: { time: 12, dish: { name: 'salad', ingredients: ['kale', 'cherry tomatoes'] } },
+  dinner: { time: 6, dish: { name: 'burrito', ingredients: ['tortilla', 'tofu', 'salsa verde'] } }
+}
+```
+
+We could return the same values from the example above like so:
+
+```ruby
+meals[:breakfast][:time]
+# => 9
+
+meals[:dinner][:dish][:name]
+# => 'burrito'
+```
+</section>
+
+There’s no limit to how nested this data can get! Gnarly!
+
+
+<section class="checks-for-understanding">
+### Checks for Understanding
 
 1. Name one common hash method and one common array method
 2. What can you ask yourself while working on nested collections to help you strategize navigating nested collections? 
+</section>
 
 
 ## Challenge
```

Comment:
The challenge below still links to the old be ruby exercises instead of the new se-exercises